nncf/quantization/algorithms/channel_alignment/openvino_backend.py

```diff
@@ -102,52 +106,34 @@ def is_node_with_bias(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
         return bias_constant is not None
 
     @staticmethod
-    def get_dims_descriptor(node: NNCFNode):
-        if node.metatype == OVConvolutionMetatype:
-            return LayoutDescriptor(
-                conv_weight_out_channels_dim=0,
-                conv_weight_in_channels_dim=1,
-                bias_channels_dim=node.metatype.output_channel_axis,
+    def get_dims_descriptor(node: NNCFNode) -> LayoutDescriptor:
+        if node.metatype in OV_CONV_METATYPES:
+            weights_layout = get_conv_weights_layout_from_node(node=node)
+        elif node.metatype == OVMatMulMetatype:
+            weights_layout = get_linear_weights_layout_from_node(node=node)
+        else:
+            raise RuntimeError(
+                f"Metatype {node.metatype} of node {node.node_name} dimensions description retrieving is not supported"
             )
-        if node.metatype in [OVGroupConvolutionMetatype, OVDepthwiseConvolutionMetatype]:
+
+        if OVConvLayoutElem.GROUPS in weights_layout:
```

Comment:
Should it work with `OVGroupConvolutionBackpropDataMetatype`?

Reply:
It should work correctly as ` get_conv_weights_layout` function supports `OVGroupConvolutionBackpropDataMetatype`